testhelper/testhelper.go

```diff
@@ -80,6 +90,25 @@ func getStubLidar(t *testing.T) *inject.Camera {
 	return cam
 }
 
+func getStubIMU(t *testing.T) *inject.MovementSensor {
```

Comment:
In an above comment I suggest a second type of invalid imu, this might be able to be used for the stub

Reply:
See my response above.